internal/templates/funcmap.go

```diff
@@ -27,11 +49,15 @@ func GetFuncMap(failMessage *string) template.FuncMap {
 		*failMessage = msg
 		return "", errors.New(msg)
 	}
+	m["formatTime"] = formatTime
 	m["toTime"] = toTime
+	m["parseTime"] = parseTime
+	m["mustParseTime"] = mustParseTime
+	m["toLayout"] = toLayout
```

Comment:
The functionality of the function is specific to times/dates, and the other new functions have this in their name. Maybe it should be called toTimeLayout / toDateLayout (or support both)?

Reply:
Could be `toTimeLayout "RFC8222"`
instead of `toLayout "time.RFC8222"`?

Reply:
Fixed with a23cb16